drivers/unix/socket_helpers.h

```diff
@@ -3,10 +3,6 @@
 
 #include <string.h>
 
-#ifndef IPV6_V6ONLY
```

Comment:
Don't you just lack an include of the Windows header that brings this definition? Judging by https://msdn.microsoft.com/en-us/library/windows/desktop/ms738574(v=vs.85).aspx I guess it may be `<ws2tcpip.h>`?